extensions/2.0/Khronos/KHR_materials_variants/schema/mapping.schema.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
The schemas need to be updated to match the new changes.

Reply:
Done.